make/test.mk

```diff
@@ -82,4 +90,23 @@ get-test-namespace: $(OUT_DIR)/test-namespace
 	$(eval TEST_NAMESPACE := $(shell cat $(OUT_DIR)/test-namespace))
 
 $(OUT_DIR)/test-namespace:
-	@echo -n "test-namespace-$(shell uuidgen | tr '[:upper:]' '[:lower:]')" > $(OUT_DIR)/test-namespace
+	@echo -n "member-operator-$(shell date +'%s')" > $(OUT_DIR)/test-namespace
+
+###########################################################
+#
+# Deploying Host Operator in Openshift CI Environment for End to End tests
+#
+###########################################################
+
+.PHONY: deploy-host
+deploy-host:
+	$(eval HOST_NS := $(shell echo -n "host-operator-$(shell date +'%s')"))
+	rm -rf /tmp/host-operator
+	# cloning shallow as don't want to maintain it for every single change in deploy directory of host-operator
+	git clone https://github.com/codeready-toolchain/host-operator.git --depth 1 /tmp/host-operator
+	oc new-project $(HOST_NS)
+	oc apply -f /tmp/host-operator/deploy/service_account.yaml
+	oc apply -f /tmp/host-operator/deploy/role.yaml
+	cat /tmp/host-operator/deploy/role_binding.yaml | sed s/\REPLACE_NAMESPACE/$(HOST_NS)/ | oc apply -f -
+	oc apply -f /tmp/host-operator/deploy/crds
```

Comment:
wouldn't be better to call the make target?

Reply:
I have written this in this repo as this way it's quite easy to follow, otherwise you need to see what's in make target in another project by changing the project